Task 6/Task6.2/REGISTRATION FORM.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Task 8.1</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+
+    <fieldset class="registration">
+
+        <h1>Регистрация</h1>
+        <hr>
+        <div class="block-registration">
+            <label for="email">E-mail</label><br>
+            <input type="email" id="email"><br>
+        </div>
+        <div class="block-registration">
+            <div>
+                <label for="password">Пароль</label> <br>
+                <input type="password" id="password">
+            </div>
+            <div>
+                <label for="password-again">Ещё раз</label><br>
+                <input type="password" id="password-again">
+            </div>
+        </div> <br>
```

Comment:
Instead of putting style like this (selector is specific about nesting, but not about what these elements are
```css
.block-registration > div > input {
    display: inline-block;
    width: 180px;
}
```

better make a class for divs that are inside `.block-registration` and give them `inline-block` and correct width, and let inputs to take a 100% of given space. To get rid of problem where inputs go beyond 100%, make `box-sizing:border-box` for every element (it's `content-box` by default)

And you don't need `<br>` here at all - all is possible using `display: block`, correct width, etc.